spec.bs

```diff
@@ -314,6 +341,31 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
 
         1. [=Assert=]: |operationMap|[|operationName|]'s [=associated realm=] is [=this=]'s [=relevant realm=].
         1. Let |operation| be |operationMap|[|operationName|], [=converted to an IDL value|converted=] to {{RunFunctionForSharedStorageSelectURLOperation}}.
+        2. Let |batchingScope| be a new [=batching scope=].
```

Comment:
nit: Let's number the step as "1." for consistency.

Reply:
Fixed!